modules/core/src/main/java/org/apache/ignite/internal/processors/odbc/jdbc/JdbcRequest.java

```diff
@@ -103,6 +106,9 @@ public class JdbcRequest extends ClientListenerRequestNoId implements JdbcRawBin
     /** Request id. */
     private long reqId;
 
+    /** Client info set with {@link JdbcThinConnection#setClientInfo} methods. */
+    private @Nullable Map<String, String> clientInfo;
```

Comment:
Only required for `JdbcQueryExecuteRequest` and `JdbcBatchExecuteRequest`. Let's introduce something like `JdbcClientInfoAwareRequest` class and inherit required classes from it.